src/main/java/de/tum/in/www1/artemis/service/AttachmentUnitService.java

```diff
@@ -90,9 +142,8 @@ public AttachmentUnit updateAttachmentUnit(AttachmentUnit existingAttachmentUnit
 
         updateAttachment(existingAttachment, updateAttachment, savedAttachmentUnit);
         handleFile(updateFile, existingAttachment, keepFilename);
-
+        existingAttachment.setVersion(existingAttachment.getVersion() != null ? existingAttachment.getVersion() + 1 : 1);
```

Comment:
```suggestion
        final int revision = existingAttachment.getVersion() == null ? 1 : existingAttachment.getVersion() + 1;
        existingAttachment.setVersion(revision);
```
Nitpick, maybe personal preference: I find it easier to read if the condition is not negated.